# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../elmer-fem/compiler.patch.front
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- front-5.2.0/include/ecif_inputFront.h.orig	2006-08-18 02:35:03.000000000 +0200
+++ front-5.2.0/include/ecif_inputFront.h	2006-08-18 02:35:37.000000000 +0200
@@ -65,7 +65,7 @@
   static bool readName(emf_ObjectData_X* od, char*& name);
   static int readVertex(emf_ObjectData_X* object_data);
   static int readVertexTable(emf_ObjectData_X* object_data);
-  static bool InputFront::storeMatcData(MatcValueTable& matcTable, const char* key, emf_ObjectData_X* od);
+  static bool storeMatcData(MatcValueTable& matcTable, const char* key, emf_ObjectData_X* od);
   static int unknownFieldMsg(emf_ObjectData_X* object_data, bool is_fatal = true);
   static int unknownObjectMsg(emf_ObjectData_X* object_data, bool is_fatal = true);
 
--- front-5.2.0/src/ecif_userinterface_TCL.cpp.orig	2006-08-18 02:39:00.000000000 +0200
+++ front-5.2.0/src/ecif_userinterface_TCL.cpp	2006-08-18 02:39:10.000000000 +0200
@@ -5018,7 +5018,7 @@
 
     //---Save data into model
     //model->setParameter(param_type, pid, oid, prtag, prtype, data_buffer, name_buffer);
-    if((int)name_buffer == 0x1) 
+    if((long)name_buffer == 0x1) 
     {
       printf("shit\n");
     } else {
